### ISSUE TITLE
Add <a href='...' sapper-noscroll>

### ISSUE THREAD

**Issue by Rich-Harris** [sveltejs/sapper#376]
Occasionally you want a link to take you to a new route without affecting the scroll position (e.g. you're flicking through some horizontal tabs that have been pushed down by a header). Currently it requires some hoop-jumping.

It would be cool if you could just do this instead:

```html
<a href="foo" sapper-noscroll>
  clicking this link won't reset the scroll position to zero
</a>
```

**Comment by arggh** [comment]
I just implemented a quick ad-hoc work-around, but it's annoyingly janky, since I have to persist the scroll position and the resume it every time the route changes. Maybe there's a better way to overcome the jank, but nevertheless this feature would be much appreciated!

**Comment by Rich-Harris** [comment]
This is implemented in 0.22.10

**Comment by elcobvg** [comment]
Just what I was looking for, thanks!
